README.md

```diff
@@ -243,6 +245,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [NewsBlur](https://github.com/samuelclay/NewsBlur) - NewsBlur is a personal news reader that brings people together to talk about the world. A new sound of an old instrument
   * [Reader-Self](http://readerself.com/) Self-hosted rss reader (php / mysql or sqlite) - Google Reader alternative ([Source code](https://github.com/readerself/readerself))
   * [Readr](https://github.com/pabloprieto/Readr) - A clean & simple, self-hosted RSS reader
+  * [RSS2EMail](http://www.allthingsrss.com/rss2email/) - exactly what you'd expect from it's name `GPL` `Python`
```

Comment:
Please a relevant description; "tool for getting news from RSS feeds in email" seems ok.